src/operations_client.ts

```diff
@@ -14,10 +14,13 @@
  * limitations under the License.
  */
 
+import {GoogleAuth} from 'google-auth-library';
```

Comment:
Just a note: one important thing about this `operations_client.ts` is that it was originally auto-generated by Veneer. Since it's very unlikely that someone will update `operations.proto`, I think it's OK to stop considering this file as autogenerated and fix whatever we want here.

Reply:
@alexander-fenster @JustinBeckwith I'm actually in the process of sending changes to operations.proto to Github rn. We should continue to treat it as autogenerated.

Reply:
As discussed offline, we can continue to manually update this file with the new changes to `operations.proto`.